Makefile

```diff
@@ -35,8 +34,13 @@ release: build
 	    echo '```' ; \
 	done
 
+.PHONY: validate
+validate: test lint
+
+.PHONY: test
```

Comment:
This `.PHONY test` should go with the test target which was added in one of the previous commits

Reply:
done :+1: